src/client/api/events/node_event.c

```diff
@@ -1,11 +1,12 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-#include "node_event.h"
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
+
 #include "client/network/mqtt/mqtt.h"
+#include "node_event.h"
```

Comment:
```suggestion
#include "client/api/events/node_event.h"
```

Reply:
I had done this but when running clang-format hook, it puts node_event.h on top of node_event.h.

Reply:
> it puts node_event.h on top of node_event.h.

I'm not sure I understand what you mean by this but the point is to include the header file with the full path, the ordering shouldn't be a problem.

Reply:
Sorry my bad, What I meant is, if I change as below :
#include "node_event.h"
#include "client/api/events/node_event.h"

and the run clang-format, it changes code to  :
#include "client/network/mqtt/mqtt.h"
#include "node_event.h"

Reply:
I think you misread the diff notation.
the suggested change is
```
#include "node_event.h"
```
to
```
#include "client/api/events/node_event.h"
```
which modify line 9 only.

> Sorry my bad, What I meant is, if I change as below :
> #include "node_event.h"
> #include "client/api/events/node_event.h"

including `node_event.h` and `client/api/events/node_event.h` looks something wrong, isn't?

Reply:
Sorry, I misinterpreted it. Yes it should be client/api/events/node_event.h.
Done.